webcomponents/tool/src/main/frontend/packages/sakai-push-utils/web-test-runner.config.mjs

```diff
@@ -0,0 +1,24 @@
+const filteredLogs = ['Running in dev mode', 'lit-html is in dev mode'];
```

Comment:
Could this be trimmed as well?

Reply:
I think we need that. Or, do you mean it just needs updating to catch "lit is in dev mode" or something?

Reply:
I said it not being sure what is needed and what not, I would be in favor of reducing the boiler plate code to a minimum. But you know better what is needed.

Reply:
I'm not sure if I know better or not ...

I know that I used to see these warnings about lit running in dev mode, and now I don't. I assume it's that filterLogs statement that has removed that warning when running tests.